src/demo/native/story.js

```diff
@@ -46,6 +46,68 @@ storiesOf('Demos', module)
 
```

Comment:
General Comment: What do you think about to generate a small React app for this? Like here `/src/components/10-atoms/button/react/demo.jsx`.

Reply:
Yeah why not. I'll do it now.